packages/server/billing/helpers/adjustUserCount.ts

```diff
@@ -150,8 +146,12 @@ export default async function adjustUserCount(
   const r = await getRethink()
   const orgIds = Array.isArray(orgInput) ? orgInput : [orgInput]
 
+  const user = await getUserById(userId)
+  if (!user) {
+    throw new Error(`User does not exist: ${userId}`)
+  }
   const dbAction = dbActionTypeLookup[type]
-  await dbAction(orgIds, userId)
+  await dbAction(orgIds, user)
```

Comment:
+1 I would prefer if we'd pass just `userId, email` to make clear that's the only thing we're using from the object. No big deal though.

Reply:
I have other refactor coming up that needs the whole user object. You can check it out later!